Source/Core/Core/HW/WiimoteReal/WiimoteReal.cpp

```diff
@@ -291,8 +291,7 @@ bool Wiimote::IsBalanceBoard()
 
     switch (InputReportID(buf[1]))
     {
-    case InputReportID::Status:
-    {
+    case InputReportID::Status: {
```

Comment:
clang-format does *not* want the brace to be on its own line? o.O

If so, could we fix the configuration? It looks inconsistent with the rest of the code style...